Resources/Prototypes/Recipes/Crafting/Graphs/storage/tallbox.yml

```diff
@@ -19,8 +19,6 @@
       conditions:
       - !type:StorageWelded
         welded: false
-      - !type:Locked
-        locked: false
```

Comment:
is this not just going in circles? shouldn't the check just not fail if the locker doesn't have a lock?

Reply:
nono, closets and lockers are different things. Closets never have a lock, lockers always have a lock.
ClosetSteelSecure (otherwise known as lockers) already had the check to deconstruct based on lock status.

The real reason that you could deconstruct lockers previously is because there are some lockers that inherit LockerBase while others inherit LockerBaseSecure. LockerBase used the ClosetSteel graph (which is supposed to only be intended for closets, not lockers) while LockerBaseSecure used the ClosetSteelSecure graph (which is correctly intended only for lockers).

I made it so that LockerBase uses the ClosetSteelSecure graph, since that is the actual intended graph for Lockers.

Reply:
to clarify further

some lockers were incorrectly using
```
   id: ClosetSteel

  description: A tall steel box that cannot be locked.
```
while some lockers were correctly using
```
  id: ClosetSteelSecure

  description: A tall steel box that can be locked.
```

and I just changed it so that all lockers were correctly using the ClosetSteelSecure graph